localization/ekf_localizer/include/ekf_localizer/pose_measurement.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Autoware Foundation
```

Comment:
Do we need this script? (Just forgot to remove?)

Reply:
I think so, as in other files

Reply:
Sorry for the ambiguity.
What I meant was about the file (=`pose_measurement.hpp`) itself.

Reply:
I forgot to remove it. Thank you for notifying

Reply:
Resolved in https://github.com/autowarefoundation/autoware.universe/pull/1795/commits/bdcb3fed2155bc48a60f5e24938ec5274e6c46db